src/features/campaigns/components/ActivityList/items/EmailListItem.tsx

```diff
@@ -0,0 +1,68 @@
+import { FC } from 'react';
+import { EmailOutlined, Person } from '@mui/icons-material';
+
+import ActivityListItemWithStats from './ActivityListItemWithStats';
+import useEmail from 'features/emails/hooks/useEmail';
+import useEmailTargets from 'features/emails/hooks/useEmailTargets';
+import ActivityListItem, { STATUS_COLORS } from './ActivityListItem';
+import useEmailState, { EmailState } from 'features/emails/hooks/useEmailState';
+
+interface EmailListItemProps {
+  emailId: number;
+  orgId: number;
+}
+
+const EmailListItem: FC<EmailListItemProps> = ({ orgId, emailId }) => {
+  const { data: email } = useEmail(orgId, emailId);
+  const state = useEmailState(orgId, emailId);
+  const { data: targets } = useEmailTargets(orgId, emailId);
+
+  if (!email) {
+    return null;
+  }
+
+  let color = STATUS_COLORS.GRAY;
+  if (state === EmailState.SENT) {
+    color = STATUS_COLORS.GREEN;
+  } else if (state === EmailState.SCHEDULED) {
+    color = STATUS_COLORS.BLUE;
+  }
```

Comment:
Minor: Perhaps an Object with keys matching the states and values that are the colors?